pyproject.toml

```diff
@@ -14,3 +14,10 @@ force_grid_wrap=0
 include_trailing_comma=true
 multi_line_output=3
 use_parentheses=true
+
+[tool.codespell]
+# Ref: https://github.com/codespell-project/codespell#using-a-config-file
+skip = '.git*'
+check-hidden = true
+# ignore-regex = ''
+# ignore-words-list = ''
```

Comment:
I don't think we need a config _yet_. Can be added in the future if needed.

Reply:
ok, removed. Note that few dotfiles/directories you have would not be checked... minor sacrifice ;)